### PR TITLE
Added code to merge two pdfs using javascript

### LINKED ISSUE #152: Script to merge PDF files
# Description

Develop a script for merging PDF files. 

## Language

- [x] Python

## Type of issue

- [x] Feature

## Checklist:

- **For Feature**

|name|about|labels|assignees|
|----|-----|------|---------|
|Merge PDF files|We have many PDF files of our course at college |Python| |

**Is your feature request related to a problem? Please describe.**: We generally have many PDF files related to the same subject of our course. So why not simply merge them into one and get rid of finding them every time we need it?

# Update

The script is open to all in Golang and JavaScript.

For python we have a script [here](https://github.com/HarshCasper/Rotten-Scripts/tree/master/Python/Merge_2_PDFs). Kindly suggest improvements then only this can be assigned to you 😄

### PR DESCRIPTION
**Pull request by harshita2319**
# Description

Added code to merge two PDFs using Javascript to the folder 'JavaScript' under the name 'Merge_pdfs'.
Also added the pdfs which I used to merge and a README file along with req screenshots.

Fixes #152 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by harshita2319** [comment]
@vybhav72954 please review!

**Review by vybhav72954 — APPROVED** [review]

File: JavaScript/Merge_pdfs/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Merging two pdf files
+
```
Comment (sohamsshah) [inline JavaScript/Merge_pdfs/README.md:2]: Can you describe the use-case of this particular script in more detail in the README?

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Great work! @harshita2319 
Reflect to the few minor changes and it is good to merged :)

Thanks!

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Hi @harshita2319 , thank you for such a swift response. You have removed the underscores from the name, but haven't yet changed them to camel case.

By that I mean: 
`empty_pdf` -> `emptyPdf`
`page_pdf2` -> `pagePdf2` and so on.

Read about camelCase [here](https://www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference)

Can you please make the small changes? Sorry for being a bit picky about naming but why not follow the best practices for such a good code-implementation😊💪

Again, thank you for working on this!

**Comment by harshita2319** [comment]
> Hi @harshita2319 , thank you for such a swift response. You have removed the underscores from the var and func names, but haven't yet changed them to camel case.
> 
> By that I mean they must be changed to:
> `empty_pdf` -> `emptyPdf`
> `page_pdf2` -> `pagePdf2` and so on.
> 
> Read about camelCase [here](https://www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference)
> 
> Can you please make the small changes? Sorry for being a bit picky about naming but why not follow the best practices for such a good code-implementation
> 
> Again, thank you for working on this!

Ya, sorry. I've updated that. Please check. Thanks! :)

**Review by sohamsshah — APPROVED** [review]
Perfect LGTM! @harshita2319

**Comment by vybhav72954** [comment]
@sohamsshah I believe we are good to go with this PR right?

**Comment by sohamsshah** [comment]
> @sohamsshah I believe we are good to go with this PR right?

Yes

**Pull request merged**